bcda/utils/common.go

```diff
@@ -72,3 +72,19 @@ func IsUUID(s string) bool {
 	re := regexp.MustCompile("^[a-fA-F0-9]{8}(?:-[a-fA-F0-9]{4}){3}-[a-fA-F0-9]{12}$")
 	return re.MatchString(s)
 }
+
+// CloseSafely will close a file and log any errors
+func CloseSafely(f *os.File) {
```

Comment:
Naming functions is hard :)
What exactly do we mean by "safely"?  And what exactly are we closing?
Recommend something like `CloseFile()`